README.md

```diff
@@ -27,10 +27,10 @@ By following the instructions for the open source version, you obtain source cod
 ### Setup with NPM
 
 ```
-npm install @telekom/scale-components-neutral@next
+npm install @telekom/scale-components-neutral
 ```
 
-> To install the [version prior to dark mode](https://github.com/telekom/scale/releases/tag/v3.0.0-beta.53), do `npm install @telekom/scale-components-neutral` (without `@next`)
+> To install the [version prior to dark mode](https://github.com/telekom/scale/releases/tag/v3.0.0-beta.53), do `npm install @telekom/scale-components-neutral@3.0.0-beta.56`
```

Comment:
@nowseemee I would replace the 53 in `v3.0.0-beta.53` with 56, everywhere there is the link. WDYT?

Reply:
we are not doing this today, im reverting